### ISSUE TITLE
No such file or directory: u'/usr/local/lib/python2.7/dist-packages/mkdocs_bootswatch/flatly/mkdocs_theme.yml'

### ISSUE THREAD

**Issue by Ezwen** [mkdocs/mkdocs-bootswatch#42]
I've installed mkdocs and bootswatch using pip:
~~~
$ sudo pip install mkdocs
$ sudo pip install mkdocs-bootswatch
~~~

Then i've configure the `flatly` theme in my mkdocs site configuration, and I end up with:
~~~
$ mkdocs -v build
DEBUG   -  Loading configuration file: /srv/admin-docs/site-markdown/mkdocs.yml 
DEBUG   -  [Errno 2] No such file or directory: u'/usr/local/lib/python2.7/dist-packages/mkdocs_bootswatch/flatly/mkdocs_theme.yml' 
~~~

And indeed, the file there does not exist.

Am I doing something wrong?

**Comment by Gwend4l** [comment]
The problem disappears if I manually create the file `/usr/local/lib/python2.7/dist-packages/mkdocs_bootswatch/flatly/mkdocs_theme.yml` with the content:
~~~
static_templates: []
~~~

**Comment by waylan** [comment]
This was fixed in #36.
